packages/cdai/src/globals/scss/_variables.scss

```diff
@@ -0,0 +1,7 @@
+// NOTE @simon See https://www.carbondesignsystem.com/guidelines/layout/#breakpoints
```

Comment:
Why do we not read breakpoint information from `/carbon/carbon/packages/layout/scss/_breakpoint.scss`?

Reply:
good question, I'll let Matt answer this one

Reply:
no idea, that's what was already there